vmdb/app/models/miq_queue.rb

```diff
@@ -441,6 +441,7 @@ def m_callback(msg, result)
 
   def requeue(options = {})
     options.reverse_merge!(self.attributes.symbolize_keys)
+    options.delete :id
```

Comment:
LOL, I was just trolling... I don't dislike this specific style with a single argument.  I always use `puts "stuff"` and leave off the parens.